File: Sorting and Searching/Reading Books/ReadingBooks.cpp
#include<cstdio>
#include<algorithm>

using namespace std;

int main(){
    int n;
    scanf("%d", &n);
    long long sum=0, m = 0, x;
    for(int i=0;i<n;i++){
        scanf("%lld", &x);
        sum += x;
        m = max(m, x);
    }
    m = max(2*m, sum);
    printf("%lld\n", m);

}

File: Sorting and Searching/Reading Books/ReadingBooks.py
import sys
from time import time
s = time()
input = sys.stdin.readline

n = int(input())
a = list(map(int, input().split()))
m = max(2*max(a), sum(a))

sys.stdout.write(str(m)+'\n')
sys.stderr.write("Time taken: "+str(time()-s)+"s\n")